code/src/lib/Button.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
This is a great approach to creating a styled component.  
One `StyledButton` which is only repsonsible for styling, and a `Button` that uses it, but contains logic.  

In this case, you don't need to export `StyledButton`  since it should only be used by `Button`